e2e-tests/tools/common-config/index.js

```diff
@@ -14,6 +14,9 @@
  * limitations under the License.
  */
 /* global jasmine */
+const path = require('path');
+// Env vars on new lines in the form of NAME=VALUE in testenv file at root of project
+require('dotenv').config({path: path.join(__dirname, '..', '..', '..', '..', 'testenv')});
```

Comment:
Any sample/library that wants to run e2e tests can now provide env vars at the root of the project in a file testenv